lightkurve/lightcurve.py

```diff
@@ -408,8 +408,8 @@ def plot(self, ax=None, normalize=True, xlabel='Time - 2454833 (days)',
             flux, flux_err = normalized_lc.flux, normalized_lc.flux_err
         else:
             flux, flux_err = self.flux, self.flux_err
-        if flux_err is None:
-            ax.plot(self.time, flux, marker='o', color=color,
+        if (flux_err is None) or (np.nansum(flux_err)==0):
```

Comment:
We can get rid of the `flux_err is None` test because the constructor now guarantees that `flux_err` is a NaN-array of length `len(time)` when `flux_err=None` is passed.

How about
```if np.any(np.isfinite(flux_err)):``